src/App.js

```diff
@@ -0,0 +1,56 @@
xAEQRCE69gRP47JGzbFO3PyBG55//tw493vRSaX8+cGI5XJ4PSdd+CeX/wFf07MeI4WPj0Or67XNxnoEiymg+ZGC5tvb+PqKxtYf2NTPa+germO1nYbri2yJwiCIAjCweyIn8dizYlwdftuuOsunLnzzr6lr0s6l8WdP38/phYXkYx5pJCE1+pE+zzTnzPeOJaLdsVEfaWBrfM1bLy5paN7lL761SactiOFlgVBEARBCMSO6bmZLLwQEb9UNovFszdhemnRn9M/CfW97ATCqGE6k/HnxoCSPRZoZk9enS8dUyhyLLlC4WOP3M23Kjqdu/ryGrbeqcKsydi5giAIgiD0zzXx0xG//sVv/swplGandZu9QbnhrvfoTiKxoIs0cwzeBhIYw/FnlYcyVeu0lfBtmVrwLn1/BZf+aUVH95pbLfW+jJsrCIIgCEJ4rolfyDTv7IkTyO8qqjoIJ2+7FUY6jlSv15E+t67Eb3za9TGyR5ljZ4zWVluncc9/+zLe/NsLutNGc70lnTQEQRAEQYiMcLYXE2wjmAgpoIeh2/W5W+pxvFKkTttFbaWJyy9cxVtfv4iVH63pGnxSYFkQBEEQhDgYK/GLBxeGtTw2kT7K3vb5Ks5/exlv/NU7uPAPy6gs1+BaksYVBEEQBCFeBhY/yzThOI7/ajBqm1twI/qsLkn7ivr/6CNotSsNHdFjZO/KD9dQW22I7AmCIAiCMFR2xK8NR+lR/4K0vbqKVjWagdLXzp+HY4dLx5pGEpaxu3OK20nvjqgHL9vmtattrP9kC+e/dRlXfnQVWxeqMCumHkbNs9XWljIsgiAIgiAMkR3xq7faSkT8F31QWV1DbXNbR/4G5e0fvgTXDhfxc5MJPXVgkeYmks72UHvwUubMqoVtJXgUPUb2Nt7aRm2lrkuz6BE1RPYEQRAEQRgRO+K3WW/ADSElZruNi6++iqvnL/hz+ofp3a2VFbzz0kuhBdI2ErAMtToeI31tfxze4RRp5qgZlL3VH68p2buKtdc6I2owvWvWTOmsIQiCIAjCWLAjfuvNeui2epffeAMXX/4xausb/pzgMALWrNXw8t99A43tipKk/iN0TPGaqSRctTYJtJX4VZT0tf1344EFlil2TOWucySN15XsvVNF9Upn+DQpwyIIgiAIwrhhnDlz5o/95zg7M4d0iALKVrutJlMXcS4vzCOdzfrvHI6rJI/tA99+8Uf44d98DbYVrn1fI5fCVimDZpb1+mpIelU1N2LxUj5qtx09akZjvanLsFQuVrGlptaWCYtpXJE9QRAEQRDGmB3xsxwHt87NIZtK6yHU+qVRqWiJSySTyE+VkUpnkGTq9QAc20ZldRUXXn4F3//KX+nfD8u2kr6tYhp2oqHEj9G+6HoGU+Yc09Ft9FhQeftSDRvntlC9XIdZtySNKwiCIAjCxLAjfmzfN53NYa5QRDKE+BHK28pbb6FVq6E4O6s/x1Vi5CmppCBR9mzThNloYu3CBbzyzW/jxb/56kC9gjm+8NpUFtW8rYSPQ7IN3q6Py8rh09jRpL3VRkXJ3tpPtrD59rZO4/I9QRAEQRCESSNx//3371jMiWIJv3TLbSik0qHG7d0N077Tiwu44b3vRTqXQb48pYWwur6B9UuXdGcO2xx8JI1KPoWLCylUs3UkXaZ4Q6K2guu4egg1RvdqVxuoXm7ArErnDEEQBEEQjgbvEj/yG3fehflCEUYMQ6fFwYWFLNaLNVhGQ70KKWiUPiV8W+er2L5YQXOjLbInCIIgCMKR4zrxu3lmDj996rRO+w4a9Yubaj6Fd+abaGSaoer1WQ2r00njUlU9UhwFQRAEQRCOLteF9S5WtrBSq6Id8dBpcbBeasIyWLYleHTObjq6zd7y91fw9tcvYfXldd1LVxAEQRAE4ahznfjZrotzmxu4Wq+FKug8LDaLNiq5Fpykg0QP8eOIGvXVJlZeWsOlf7qiZa9ysaZ75XL4NOmsIQiCIAjCceC6VC9JJ5O4ZXYedy4sYaFQ8OeOD2bKxRtLDTSyDrzE/tLGNnocF7e21kRzowWr6cBqmLBbnR7GgiAIgiAIx419xY8U0xncMb+gpkUUMxl/7uhxlehdnGthZfr6kTlYc4+FlFuVlh4zt7VtorXVRrs6nKHbBEEQBEEQxpkDxY+wg8cts3O4bW4eZXb2GDGW4WKjZOHCXBOe3++E0TvKHkuwsMaefqy0YdWZwu2/w4cgCIIgCMJR5VDxI5S/2+YWcMfCAvKs7zcCqG9m2kE1Z2N5tq1TvVbTgt100a52InqNjZYutuxYInuCIAiCIAj70VP8CNO+9yydxJnpGUxlMqGGdAuLq76qnXKwUbSwVmqjZplw2g7qV5t6CLX6elMPqSYIgiAIgiAcTiDxI1nDwNmZeSWAJ1DKZmEMQf7Ynq+WdrCWa2M1q0SvbqGyXEX1Sk2XZREEQRAEQRCCE1j8CMfeLaQzePCmm7FYLsNAfPLnKOnbyJu44FVxaW0L1eUazNrgQ7wJgiAIgiAcV/oSv91whI+7FhaxWFICGFH0jwvCmnyXGzWca2/gwtVNtOsie4IgCIIgCFEQWvy6nCiWcNPMLG6Ymu4M80b6EEF+ued5aFgm3lhfw7nKOuotE67U2hMEQRAEQYiUgcWP6d9U0kDWSOq2f/P5AhYKJZQzWeQyad0xZC91JXmVdgv1tno0W3qUkO1WW48aYrmdtnuUQUEQBEEQBCE6Bha/LozxpZJJpAwDGTWlEgaSycS+aWBHSZ2tBM/RoufCtG09TxAEQRAEQYiPyMRPEARBEARBGG+S/qMgCIIgCIJwxBHxEwRBEARBOCaI+AmCIAiCIBwTRPwEQRAEQRCOCSJ+giAIgiAIxwQRP0EQBEEQhGOCiJ8gCIIgCMIxQcRPEARBEAThmCDiJwiCIAiCcEwQ8RMEQRAEQTgmiPgJgiAIgiAcE0T8BEEQBEEQjgXA/w/xfS7gJ03WxAAAAABJRU5ErkJggg=="
```

Comment:
Any specific reason to use images in b64 formats? Maybe to update from inside Dynamo?

Reply:
Can we put this embeded image in a different js file, I am worried this big block will affect local dev and PR review in the future

Reply:
Done, Created a new js file called `encodedImages` containing the long strings